src/redux/userStats/reducer.ts

```diff
@@ -0,0 +1,82 @@
+import { Reducer } from 'redux';
+import { UserStatsAction } from './index';
+
+export interface UserContactEvent {
+  contactId: string;
+  issueId: string;
+  result: string;
```

Comment:
If result options become an enum, then the result property of `UserContactEvent` will be that enum type instead of string.